protocol/0088-TRTO-trading_transaction_ordering.md

```diff
@@ -0,0 +1,21 @@
+# Trading Transaction Ordering
+
+In order for an exchange to offer competitive prices for parties arriving and wishing to trade immediately, others have to be willing to place limit orders on the book which will remain there and wait for an incoming order to match with it. These are required for a limit order book to exist (as implied by the name) but expose the party placing the initial limit order to a range of risks, as often the reason these orders will eventually trade is because the price has become unfavourable to the party who placed the order, but they either have not realised or have not had a chance to cancel the order yet. This is often referred to as "toxic flow". Whilst another party obviously gains from this transaction, it is generally acknowledged that the higher a given venue's proportion of toxic flow to more balanced flow, the wider market makers will end up quoting to avoid being the victim of this. As such, exchange and transaction designs which allow for the reduction of this without unfairly impacting other parties using the network may allow for the exchange as a whole to provide traders with better prices than otherwise. This specification covers one such proposed methodology, comprising of a specified ordering of order executions on a given market.
+
+## Execution Ordering
+
+Trading transactions (those which create, update or remove orders of any type on any market) should be executed in a specific way once included in a block.
```

Comment:
We should probably add that the re-ordering only matters on a per-market basis and furthermore it should be possible to be enabled on a per-market basis.